style.css

```diff
@@ -928,8 +2035,158 @@ body {
     top: 614px;
   }
 
+ .second__wrap {
+    margin: 0;
+  }
+
+  .second-page {
+    flex-direction: column;
+  }
+
+  .second-page__img {
+    margin-right: 0;
+    margin-bottom: 4%;
+  }
+
+  .second__img_purses {
+    width: 67%;
+    padding: 12% 0 0 17%;
+  }
+
+  .second__wrap {
+    margin: 0 0 80px 0;
+  }
+
+  .second-page__article {
+    width: 355px;
+    margin-bottom: 0;
+  }
+
+  .second-page__title {
+    font-size: 30px;
+  }
+
+  .second-page__subtitle {
+    font-size: 15px;
+    letter-spacing: 1px;
+    margin-bottom: 20px;
+  }
+
+  .second-page__description {
+    font: 600 12px/16px 'Montserrat-Regular', sans-serif;
+    margin-bottom: 24px;
+    letter-spacing: -.4px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 30px;
+  }
+
+  .eclipse {
+    width: 25px;
+    height: 25px;
+    margin-right: 9px;
+  }
+
+  .second-page__price {
+    margin-bottom: 30px;
+    font-size: 30px;
+    margin-right: 46px;
+  }
+
+  .second__quantity__input {
+    font-size: 15px;
+    padding: 7px 0;
+    width: 20px;
+  }
+
+  .second__add-to-cart {
+    font-size: 14px;
+    width: 245px;
+    padding: 10px 16px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 147px, #000 147px, #000 194px, #EBE3D8 194px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)